ui/libs/__init__.py

```diff
@@ -19,11 +21,24 @@ class Token:
     name: str
     address: str = None
 
+    def __str__(self):
+        return self.name
+
+
+@dataclass
+class EVM:
```

Comment:
bad name

Reply:
Changed it to `Platform`, I hope it better reflects the meaning, but let me know if there's better naming.